config/govuk_pages.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
No, its been removed in the latest version

On 21 December 2015 at 09:43, Tijmen Brommet <notifications@github.com>
wrote:

> In config/govuk_pages.yml
> <https://github.com/alphagov/govuk-diff-pages/pull/1#discussion_r48129585>
> :
>
> > @@ -0,0 +1,41 @@
> > +---
>
> Should this file be checked in? It's auto-generated when the differ runs,
> so it might not be?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/govuk-diff-pages/pull/1/files#r48129585>.
>



-- 
Stephen Richards
stephen.richards@digital.cabinet-office.gov.uk